### DISCUSSION TITLE
Missing download files

### DISCUSSION THREAD

**Discussion by craig-wallin** [geneontology/helpdesk#492] [Q&A] [answered]
Hi - Recently I discovered that 15 files that were formerly available for download have gone missing.  Specifically, these files:

goa_chicken_complex.gaf.gz
goa_chicken_isoform.gaf.gz
goa_chicken_rna.gaf.gz
goa_cow_complex.gaf.gz
goa_cow_isoform.gaf.gz
goa_cow_rna.gaf.gz
goa_dog_complex.gaf.gz
goa_dog_isoform.gaf.gz
goa_dog_rna.gaf.gz
goa_human_complex.gaf.gz
goa_human_isoform.gaf.gz
goa_human_rna.gaf.gz
goa_pig_complex.gaf.gz
goa_pig_isoform.gaf.gz
goa_pig_rna.gaf.gz

Is this expected?  Have they been moved elsewhere?

Thanks,
Craig

**Comment by kltm** [comment, marked as answer]
@craig-wallin Actually, this is expected (looping in @pgaudet if further details are needed). Essentially, the contents of these files were folded into the "root" file. So, for example, 
```
goa_pig_complex.gaf.gz
goa_pig_isoform.gaf.gz
goa_pig_rna.gaf.gz
goa_pig.gaf.gz
```
and now just
```
goa_pig.gaf.gz
````

**Comment by pgaudet** [comment]
This is correct.
